src/io/flutter/preview/PreviewView.java

```diff
@@ -843,6 +847,34 @@ private void scrollTreeToNodeOnTop(TreeNode node) {
     }
   }
 
+  private void showNotRenderableInPreviewArea(@NotNull FlutterOutline widget) {
+    final JPanel panel = new JPanel();
+    panel.setLayout(new MigLayout("", "[grow, center]", "[grow][][][grow 200]"));
```

Comment:
I'm not familiar with this layout. One suggestion: for a simple layout, consider using one that is  familiar to more devs on the project - like nested panels with BorderLayouts. For complicated layouts using whatever tool is best sgtm.

Reply:
I looked for standard Swing layouts and only GridBagLayout works well for it.
But code is so cumbersome, that I'd prefer to stay with MigLayout.